aten/src/ATen/native/native_functions.yaml

```diff
@@ -37,19 +37,31 @@
   use_c10_dispatcher: full
   variants: function
 
+# Computes the gradient of current tensor w.r.t. graph leaves.
 - func: backward(Tensor self, Tensor? gradient=None, bool keep_graph=False, bool create_graph=False) -> ()
   variants: method
 
+# DEPRECATED. Sets the tensor data held by this `Variable` to be the same as
+# `new_data`.  It requires that `new_data` and `Variable` have compatible tensor
+# type, by checking `_has_compatible_shallow_copy_type(this, new_data)`.
+#
+# This function is deprecated because it doesn't really make sense in a world
+# where Variables *are* Tensors (as opposed to them containing tensors, which
+# is what the previous interpretation was.)
 - func: set_data(Tensor(a!) self, Tensor new_data) -> ()
   use_c10_dispatcher: unboxed_only
   variants: method
 
 - func: data(Tensor self) -> Tensor
   variants: method
 
+# True if this `Variable` is a leaf and thus does not have a `grad_fn`.
 - func: is_leaf(Tensor self) -> bool
   variants: method
 
+# Returns the input index of the gradient `Node` to which this
+# `Variable` is connected.  Note: input indexes of the gradient `Node`
+# correspond to output indexes of the corresponding forward `Node`.
```

Comment:
"forward `None`" is not really a thing :/
Maybe just "forward function" here?

Reply:
I plead preexisting comments XD (ok will fix)